.github/ISSUE_TEMPLATE/generic-issue-template.md

```diff
@@ -1,32 +1,44 @@
 ---
 name: Generic Issue Template
 about: This template is for any new issue
-title: DR-
-labels: ''
-assignees: ''
-
+title: "DR-{{ random }}"
```

Comment:
This is an attempt to see if the issue number will attacheck when creating the issue